src/ingest/build.sbt

```diff
@@ -0,0 +1,47 @@
+import Dependencies._
+
+name := "ingest"
+
+// dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-core" % "2.6.7"
+// dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-databind" % "2.6.7"
+// dependencyOverrides += "com.fasterxml.jackson.module" % "jackson-module-scala_2.11" % "2.6.7"
+
+libraryDependencies ++= Seq(
+  decline,
+  spark % "provided",
+  sparkSql % "provided",
+  sparkJts,
+  gtShapefile exclude("com.google.protobuf", "protobuf-java"),
+  "com.google.protobuf" % "protobuf-java" % "2.5.0",
+  vectorpipe exclude("com.google.protobuf", "protobuf-java"),
+  gtRaster,
+  gtS3,
+  cats,
+  scalactic,
+  scalatest,
+  "com.amazonaws" % "aws-java-sdk-s3" % "1.11.340" % "provided"
```

Comment:
Maybe incorporate this approach instead? https://github.com/azavea/osmesa/issues/159

(or at least see if it's relevant here; EMR compatibility was the initial reason for setting `provided`. I also seem to remember having to do some digging to figure out version numbers for AWS-related dependencies, which excluding by wildcard would simplify tremendously.)